runtime-light/stdlib/stdlib.cmake

```diff
@@ -22,6 +22,6 @@ prepend(
   rpc/rpc-tl-request.cpp
   string/string-state.cpp
   system/system-state.cpp
-  file/file-stream-state.cpp
+  resource/resource-state.cpp
```

Comment:
Not a big deal, but can you keep this list lexicographically ordered?

Reply:
fix